src/index/base.cpp

```diff
@@ -70,21 +70,21 @@ bool BaseIndex::Init()
     if (!m_synced) {
         bool prune_violation = false;
         if (!m_best_block_index) {
-            // it looks like this index is not built yet
-            // make sure we have all data back to the genesis
+            // index is not built yet
+            // make sure we have all block data back to the genesis
```

Comment:
d2ab62fede34b7980863617f3669c019ee42d029 

These comment were added in the previous commit, could fixup these hunks?